test.sh

```diff
@@ -164,6 +164,10 @@ GO111MODULE=off test_project gopath/src/project 1
 # test project v2
 GO111MODULE=on test_project project-v2 2
 GO111MODULE=on test_project project-v2-multigroup 2
+GO111MODULE=on test_project project-v2-addon 2
 
-exit $rc
+# test project v3
+GO111MODULE=on test_project project-v3 3-alpha
+GO111MODULE=on test_project project-v3-multigroup 3-alpha
```

Comment:
Where is the  project-v2-addon? Will it be removed?

Reply:
You mean for v3?